schemas/2015-08-01/Microsoft.Web.json

```diff
@@ -0,0 +1,389 @@
+                            "description": "Microsoft.Web/sites/extensions: parameters"
+                        }
+                    }
+                }
+            },
+            "required": [
+                "type",
+                "apiVersion",
+                "properties"
+            ]
+        },
+        "web": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "pattern": "web"
+                },
+                "properties": {
+                    "type": "object",
+                    "properties": {
+                        "phpVersion": {
+                            "type": "string",
+                            "description": "Microsoft.Web/sites/config: PHP version (an empty string disables PHP)."
+                        },
+                        "netFrameworkVersion": {
+                            "type": "string",
+                            "description": "Microsoft.Web/sites/config: The .Net Framework version."
+                        }
+                    }
+                }
+            }
+        },
+        "connectionstrings": {
+            "type": "object",
+            "properties": {
+                "name": {
+                    "pattern": "connectionstrings"
+                },
+                "properties": {
+                    "type": "object",
+                    "additionalProperties": {
+                        "type": "object",
+                        "required": [
+                            "value",
+                            "type"
+                        ],
+                        "properties": {
+                            "value": {
+                                "type": "string",
+                                "description": "Microsoft.Web/sites/config/connectionstrings: Connection string to database."
+                            },
+                            "type": {
+                                "oneOf": [
+                                    {
+                                        "$ref": "http://schema.management.azure.com/schemas/2015-05-01/deploymentTemplate.json#/definitions/expression"
+                                    },
+                                    {
+                                        "enum": [
+                                            "MySql",
+                                            "SQLServer",
+                                            "SQLAzure",
+                                            "Custom"
+                                        ]
+                                    }
+                                ],
+                                "description": "Microsoft.Web/sites/config/connectionstrings: Type of database."
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "appsettings": {
+            "type": "object",
+            "properties": {
+                "name": {
```

Comment:
Technically not required, but probably good to have it in there so users have the right values. I'll change it